command `sudo salt \* pkg.install salt-minion -p -b 3 refresh=True` fails hard:

```
Executing run on ['hey-01', 'zz-01', 'mmm']

[ERROR   ] An un-handled exception was caught by salt's global exception handler:
TypeError: output() takes exactly 2 arguments (1 given)
Traceback (most recent call last):
  File "/usr/bin/salt", line 10, in <module>
    salt_main()
  File "/usr/lib/python2.7/dist-packages/salt/scripts.py", line 458, in salt_main
    client.run()
  File "/usr/lib/python2.7/dist-packages/salt/cli/salt.py", line 98, in run
    for res in batch.run():
  File "/usr/lib/python2.7/dist-packages/salt/cli/batch.py", line 208, in run
    self.opts)
  File "/usr/lib/python2.7/dist-packages/salt/output/__init__.py", line 86, in display_output
    display_data = try_printout(data, out, opts)
  File "/usr/lib/python2.7/dist-packages/salt/output/__init__.py", line 39, in try_printout
    return get_printout(out, opts)(data).rstrip()
TypeError: output() takes exactly 2 arguments (1 given)
Traceback (most recent call last):
  File "/usr/bin/salt", line 10, in <module>
    salt_main()
  File "/usr/lib/python2.7/dist-packages/salt/scripts.py", line 458, in salt_main
    client.run()
  File "/usr/lib/python2.7/dist-packages/salt/cli/salt.py", line 98, in run
    for res in batch.run():
  File "/usr/lib/python2.7/dist-packages/salt/cli/batch.py", line 208, in run
    self.opts)
  File "/usr/lib/python2.7/dist-packages/salt/output/__init__.py", line 86, in display_output
    display_data = try_printout(data, out, opts)
  File "/usr/lib/python2.7/dist-packages/salt/output/__init__.py", line 39, in try_printout
    return get_printout(out, opts)(data).rstrip()
TypeError: output() takes exactly 2 arguments (1 given)
```
